backend/src/main/java/com/google/sps/servlets/SearchMatchServlet.java

```diff
@@ -37,6 +37,16 @@
 @WebServlet("/search-match")
 public class SearchMatchServlet extends HttpServlet {
 
+  /** Datastore Key constant string */
+  private static final String KEY_MATCH = "Match";
+
+  /** Datastore Property constant strings */
```

Comment:
tiny nit: this will show up in docs pages / editor docs as only the documentation for PROPERTY_FIRSTPARTICIPANT but not the other properties. This would be a case where you should use // instead of /** */ since this is a general comment and not a single item's javadocs